stanford-earth/matson
# EARTH-1059: Video Header Prob

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/EARTH-1059_
Fixes margin with video

# Needed By (Date)
- Soon

# Urgency
- Fixed on prod, need to put into code.

# Steps to Test

- Checkout this branch
- May need to revert feature
- Navigate to __https://earth.stanford.edu/news/double-edged-sword-palm-oil#gs.p9fsje__
- Set browser width less than 767px
- Verify you see this rule:
```
 .hero-banner.component-centered-container-margins.has-video .hero-banner__inner .hero-banner__container {
      margin: 0;
```

# Affected Projects or Products
- Earth (https://earth.stanford.edu)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/EARTH-1059




# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
